projects/ADuCM3029_demo_cn0414/src/ad717x.c

```diff
@@ -0,0 +1,505 @@
+/***************************************************************************//**
```

Comment:
I think the commit message should explain that this driver was already existing, and you haven't modified it, or maybe you did? If you didn't, it can be ignored by the reviewers :)